### PR TITLE
Fixes issue #4566 Enabling code folding by default

### PR DESCRIPTION
**Pull request by mkrueger**
https://github.com/mono/monodevelop/issues/4566

Should match the VS.NET default options.

### CODE REVIEW

**Comment by mhutch** [comment]
Will this change it for existing users?

Also this change was contingent on the hover behavior.

**Comment by mhutch** [comment]
(also, any chance we could do the fade in / fade out?)

**Comment by slluis** [comment]
Can you make a screencast of the new behavior?

File: main/src/addins/MonoDevelop.SourceEditor2/Mono.TextEditor.Theatrics/Stage.cs [hunk 1]
```diff
@@ -80,14 +80,14 @@ public Actor<T> Add (T target)
             }
         }
 
-        public Actor<T> Add (T target, uint duration)
+        public Actor<T> Add (T target, uint duration, double percantage = 1.0)
```
Comment (Therzok) [inline main/src/addins/MonoDevelop.SourceEditor2/Mono.TextEditor.Theatrics/Stage.cs]: typo: percentage

**Comment by mkrueger** [comment]
When it's merged and shipped it will affect users which installed that version.

**Comment by iainx** [comment]
Just a heads up, this has broken accessibility both keyboard and VoiceOver, because the fold markers need to be drawn if the fold margin is focused by the keyboard, or if VoiceOver is operational.

**Comment by slluis** [comment]
@iainx can you please file a bug report for this?

**Comment by iainx** [comment]
@slluis done

**Pull request merged**